packages/rule-babel-config-is-valid/README.md

```diff
@@ -0,0 +1,32 @@
+# babel-config-is-valid (`babel-config-is-valid`)
```

Comment:
Complete documentation. You can take a look at what @sarvaje is doing for TypeScript in #847